db/migrate/20240322071629_convert_cached_label_list_to_text.rb

```diff
@@ -0,0 +1,32 @@
+class ConvertCachedLabelListToText < ActiveRecord::Migration[7.0]
+  def up
+    change_column :conversations, :cached_label_list, :text
+  end
+
+  def down
+    # This might cause data loss if the text is longer than 255 characters
+    # lets start by truncating the data to 255 characters
+    Conversation.where('LENGTH(cached_label_list) > 255').find_in_batches do |conversation_batch|
```

Comment:
This would timeout.

Reply:
In production we would just have to do this manually, but for any other self hosted instances this would work fine. Besides this is just the down query, I don't think we would need to do this a lot. Let me know if you want me to move this out as a script?

Reply:
@scmmishra, this should be fine. lets go ahead with merging this.